development-docs/TESTING.md

```diff
@@ -90,19 +91,19 @@ Cluster Operator setup example:
     }
 ```
 
-##### Exercise
-In this phase you specify all the steps which you need to cover some functionality. 
-If you didn't create the Kafka cluster you should do so at the begging of test using the test method resources instance of `Resources` inherited from `BaseST`.
+### Exercise
+In this phase you specify all steps which you need to execute to cover some specific functionality.
+If you didn't create the Kafka cluster you should do so at the beginning of the test using a test method resources instance of `Resources` inherited from `AbstractST`.
```

Comment:
I don't thin this is valid now. We use two stacks and you can everywhere use static methods from "resources" classes (like `KafkaResources.class`) and `ResourceManager.class`will handle the rest. This approach in TESTING.md is quite old and I thing I just forgot to remove it.

Reply:
ok I am removing this line 96.